Automattic/vip-go-mu-plugins
# Removes elasticpress menu links

## Description
We already have some code hiding ElasticPress in general admin, but it is not there in multisite env.
This PR solves that.

![image](https://user-images.githubusercontent.com/19240162/94827448-bce05500-0408-11eb-91c7-aeaa573f5895.png)


## Checklist

Please make sure the items below have been covered before requesting a review:

- [/] This change works and has been tested locally (or has an appropriate fallback).
- [/] This change has relevant unit tests (if applicable).
- [/] This change has relevant documentation additions / updates (if applicable).

## Steps to Test

* Run your local environment or sandbox with master code
* Make sure it is a multisite
* Set following in your config:
```
define( 'VIP_ENABLE_VIP_SEARCH', true );
define( 'EP_IS_NETWORK', true );
```
* See that the menu links are there
* Checkout the branch
* Check that the menus from the screenshot are gone.
